_data/locales/ja-JP.yml

```diff
@@ -0,0 +1,91 @@
+# The layout text of site
+
+# ----- Commons label -----
+
+layout:
+  post: 投稿
+  category: カテゴリー
+  tag: タグ
+
+# The tabs of sidebar
+tabs:
+  # format: <filename_without_extension>: <value>
+  home: ホーム
+  categories: カテゴリー
+  tags: タグ
+  archives: アーカイブ
+  about: このサイトについて
+
+# the text displayed in the search bar & search results
+search:
+  hint: 検索
+  cancel: キャンセル
+  no_results: 該当なし
+
+panel:
+  lastmod: 最近更新された投稿
+  trending_tags: トレンドのタグ
+  toc: コンテンツ
+
+copyright:
+  # Shown at the bottom of the post
+  license:
+    template: この投稿は投稿者によって :LICENSE_NAME の下でライセンスされています。
+    name: CC BY 4.0
+    link: https://creativecommons.org/licenses/by/4.0/
+
+  # Displayed in the footer
+  brief: Some rights reserved.
+  verbose: >-
+    Except where otherwise noted, the blog posts on this site are licensed
+    under the Creative Commons Attribution 4.0 International (CC BY 4.0) License by the author.
+
+meta: :PLATFORM 用の :THEME を使用しています。
+
+not_found:
+  statement: このURLは存在しないものを指し示しています。
+
+notification:
+  update_found: 新しいバージョンが利用可能です。
+  update: 更新
+
+# ----- Posts related labels -----
+
+post:
+  written_by: 投稿者
+  posted: 投稿日
+  updated: 更新日
+  words: 語
+  pageview_measure: 回閲覧
+  read_time:
+    unit: 分
+    prompt: で読めます
+  relate_posts: さらに読む
+  share: シェア
+  button:
+    next: 次
+    previous: 前
+    copy_code:
+      succeed: コピーしました
+    share_link:
+      title: リンクをコピー
+      succeed: リンクをコピーしました
+
+# Date time format.
+# See: <http://strftime.net/>, <https://day.js.org/docs/en/display/format>
+df:
+  post:
+    strftime: "%Y/%m/%d"
+    dayjs: "YYYY/MM/DD"
+  archives:
+    strftime: "%b"
+    dayjs: "MMM"
```

Comment:
```suggestion
```

Except for `en`, no other locale needs this setting.

Reply:
Thank you. I deleted the lines.